docs/builtins/Operators/floor.md

```diff
@@ -0,0 +1,49 @@
+## floor
+Use `floor` to round down the value of a decimal `X` to an integer, or to a specified precision `PREC` as a decimal.
+
+### Basic syntax
+
+To round down a decimal value to an integer or to a specified precision, use the following syntax:
+
+`(floor X)`
+
+or
+
+
+`(floor X PREC)`
+
+### Arguments
+
+Use the following arguments to specify the decimal value and precision for the `floor` Pact function:
+
+| Argument | Type     | Description                                          |
+|----------|----------|------------------------------------------------------|
+| `X`        | `decimal`  | Specifies the decimal value to round down.           |
+| `PREC`     | `integer`  | Specifies the precision for the rounding (optional). |
```

Comment:
Optional arguments are either denoted as, well, optional at the end (like here) or at the beginning (`ceiling` being an example) or not at all (`read-keyset` is kinda sorta like that). It'd be great to have a more uniform way of denoting optional args.

Reply:
Agree. @lsgunnlsgunn this one too, we should create some workbook